kolibri/plugins/facility/assets/src/views/ManageClassPage/ClassCreateModal.vue

```diff
@@ -48,16 +49,12 @@
     },
     computed: {
       duplicateName() {
-        const index = this.classes.findIndex(
+        return !!this.classes.find(
```

Comment:
I prefer the explicit coercion to `Boolean` over the `!!`
```suggestion
        return !!this.classes.find(
```

Reply:
But I see we're not doing this consistently in the codebase! I'll make a note of this to make `Boolean()` the standard way to do coercion.

Reply:
I prefer the explicit version (`Boolean`) also, but i wanted to stay consistent.

I also changed the `.findIndex` to `.find` because it is more straightforward, i think. :)